src/lib/markbind/src/parser.js

```diff
@@ -205,7 +205,23 @@ Parser.prototype._preprocess = function (node, context, config) {
     let fileContent = self._fileCache[actualFilePath]; // cache the file contents to save some I/O
     const { parent, relative } = calculateNewBaseUrls(filePath, config.rootPath, config.baseUrlMap);
     const userDefinedVariables = config.userDefinedVariablesMap[path.resolve(parent, relative)];
-    fileContent = nunjucks.renderString(fileContent, userDefinedVariables);
+    const includedVariables = context.includedVariables || {};
+
+    if (element.children) {
+      element.children.forEach((child) => {
+        if (child.name !== 'span') {
+          return;
+        }
+        if (!child.attribs.id) {
+          // eslint-disable-next-line no-console
+          console.warn(`Missing 'id' in variable for ${element.attribs.src} include.`);
```

Comment:
Can also include the name of the current file? `\nMissing reference in ${element.attribs[ATTRIB_CWF]}`.

This is so that the author has an easier time finding the offending variable.